packages/front-end/components/Experiment/SinglePage.tsx

```diff
@@ -784,21 +790,50 @@ export default function SinglePage({
                 <StatusIndicator
                   archived={experiment.archived}
                   status={experiment.status}
-                  newUi={true}
                 />
               </div>
               {experiment.status === "stopped" && experiment.results && (
                 <div
                   className="d-flex border-left"
                   style={{ height: 30, lineHeight: "30px" }}
                 >
-                  <ResultsIndicator results={experiment.results} newUi={true} />
+                  <ResultsIndicator results={experiment.results} />
                 </div>
               )}
             </div>
           </div>
         </div>
 
+        {!hideNewExperimentHelp ? (
+          <div
+            className="d-flex justify-content-end small mt-1"
+            style={{ marginBottom: -10 }}
+          >
+            <div
+              className="text-gray py-1 rounded text-center d-flex align-items-center justify-content-center"
+              style={{ backgroundColor: "#f993", width: 530 }}
```

Comment:
Not sure about a red background. Makes it look like an error.

Reply:
Sure, let's go with periwinkle
<img width="675" alt="image" src="https://github.com/growthbook/growthbook/assets/7927873/c902b035-4c7d-4be8-b0b7-72d1b8ed6845">